.github/workflows/run_cell-type-wilms-tumor-14.yml

```diff
@@ -47,16 +47,33 @@ jobs:
       - name: Set up pandoc
         uses: r-lib/actions/setup-pandoc@v2
 
+      - name: Install system dependencies
+        run: |
+          sudo apt-get install -y \
+            jags \
+            libcurl4-openssl-dev \
+            libfribidi-dev \
+            libglpk40 \
+            libharfbuzz-dev \
+            libhdf5-dev \
+            libmagick++-dev \
+            libtiff5-dev
+
       - name: Set up renv
         uses: r-lib/actions/setup-renv@v2
         with:
           working-directory: ${{ env.MODULE_PATH }}
 
       # Update this step as needed to download the desired data
-      - name: Download test data
-        run: ./download-data.py --test-data --format SCE
+      - name: Download test data and results
+        run: |
+          ./download-data.py --test-data --projects SCPCP000014 --format SCE
+          ./download-results.py --test-data --projects SCPCP000014 --modules doublet-detection
 
       - name: Run analysis module
         run: |
           cd ${MODULE_PATH}
-          # run module script(s) here
+          # run individual scripts for now, until we have a main script
+          Rscript scripts/00_preprocessing_rds.R
+          # anchor transfer currently fails with a small number of cells
+          # Rscript scripts/01_anchor_transfer_seurat.R
```

Comment:
I updated this test workflow file to run the current versions of your scripts, but the anchor transfer script is currently failing with the following error:

```
Error in `map()`:
ℹ In index: 1.
Caused by error in `FindWeights()`:
! Number of anchor cells is less than k.weight. Consider lowering k.weight to less than 37 or increase k.anchor.
Backtrace:
     ▆
  1. ├─purrr::walk(...)
  2. │ └─purrr::map(.x, .f, ..., .progress = .progress)
  3. │   └─purrr:::map_("list", .x, .f, ..., .progress = .progress)
  4. │     ├─purrr:::with_indexed_errors(...)
  5. │     │ └─base::withCallingHandlers(...)
  6. │     ├─purrr:::call_with_cleanup(...)
  7. │     └─global .f(.x[[i]], ...)
  8. │       └─global run_anchorTrans(...)
  9. │         └─Seurat::TransferData(...)
 10. │           └─Seurat:::FindWeights(...)
 11. │             └─base::stop(...)
 12. └─base::.handleSimpleError(...)
 13.   └─purrr (local) h(simpleError(msg, call))
 14.     └─cli::cli_abort(...)
 15.       └─rlang::abort(...)
Execution halted
```

I suspect that the error here is due to the fact that our test data only has 100 cells, but it will take some time to test the best way to get around this error.

Separately, the current tests take a long time because of the need to download and process the large reference file. Do you know if there might be another small reference file that we could use for testing only? If not, we might consider creating a subset of the reference for testing. 

For the moment, I have commented out the anchor transfer step, but we will probably want to try to solve this as part of this PR

Reply:
I have reproduced (and solved!) this locally, so I will add comments to relevant spots of files in a review shortly.